project.clj

```diff
@@ -101,7 +101,7 @@
                        :dependencies [[puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]]}
              :ci {:plugins [[lein-pprint "1.1.1"]]}}
 
-  :test-selectors {:default (complement :integration)
+  :test-selectors {:default (constantly :all)
                    :integration :integration
                    :unit (complement :integration)
                    :all (constantly true)}
```

Comment:
Since we're reverting back to the default behaviors of leiningen, we can just get rid of the `:default` and `:all` selectors altogether, leaving us with just:
```
  :test-selectors {:integration :integration
                   :unit (complement :integration)}
```

I verified this behaves exactly as specifying all four of them in all of the cases: `lein test`, `lein test :all`, `lein test :unit`, and `lein test :integration`.